boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -84,6 +84,10 @@ The FRDM-MCXN947 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | USDHC     | on-chip    | sdhc                                |
 +-----------+------------+-------------------------------------+
+| VREF      | on-chip    | REGULATOR                           |
```

Comment:
nit: in case you have to push again for some reason, change regulator to lowercase please